scylla-server/src/services/systems.services.ts

```diff
@@ -0,0 +1,15 @@
+import prisma from '../prisma/prisma-client';
+import { ResponseFunction } from '../utils/message-maps.utils';
+
+// CRUD operation to get all systems with ResponseFunction type
+
+export const getAllSystems: ResponseFunction = async (data: JSON) => {
+  try {
+    const data = await prisma.system.findMany();
+    // was getting type errors trying to return just data
+    // so added JSON parse and stringify to make it conform to JSON
+    return JSON.parse(JSON.stringify(data));
```

Comment:
ok i didnt realize you would have to do this, so instead can you change the ResponseFunction type to return a string rather than a JSON Object and then just return JSON.stringify(data)